app/views/sidebar/_post_button.html.erb

```diff
@@ -1,10 +1,22 @@
+<% if params[:controller] == 'questions' %>
```

Comment:
Ok Jeff I will add the **Ask a question** button at the top of the question show page and index page.